packages/nuxt/src/app/composables/asyncData.ts

```diff
@@ -112,6 +118,10 @@ export interface AsyncDataExecuteOptions {
 
   /** @internal */
   cachedData?: any
+
+  signal?: AbortSignal
+
+  timeout?: number
```

Comment:
we should probably not add additional options to `execute` - what about passing an AbortController when creating useAsyncData in the first place?

(btw, there are lots of places we call `execute` at the moment, where we do not proxy the signal/timeout from the parent options)

Reply:
> AbortController signal is aborted, it cannot be restored without recreating the controller. (Maybe i am wrong?)

:+1: Yes once an AbortController is aborted, we need another one. That's why we need to have one per handler call